docs/src/main/tut/contravariant.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title:  "Contravariant"
+section: "typeclasses"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/functor/Contravariant.scala"
```

Comment:
Strange, but currently `useful links` section, with those `source` links is hidden... 
I think this issue and all links should be fixed in another pr. I'll take care of that.